src/Pages/SignIn/Components/Input.js

```diff
@@ -0,0 +1,21 @@
+import React, { useState } from "react";
+
+function Input(props) {
+  const { type, placeholder, onChange, name } = props;
+  const [inputText, setInput] = useState({ value: "" });
+
+  const pushOnChange = e => {
+    onChange({ value: e.target.value, name: e.target.name });
+    setInput({ value: e.target.value });
+  };
```

Comment:
```suggestion
  const pushOnChange = e => {
    const { value, name } = e.target
    
    onChange({ value, name });
    setInput({ value });
  };
```
- 객체 구조 분해 할당 적용
- 프로퍼티 값 단축 구문(property value shorthand) - key value 동일한 경우

Reply:
감사합니다!!